src/utils/githubApi.ts

```diff
@@ -69,6 +72,44 @@ export class GithubRemote {
         : '';
     return this.PROTOCOL_PREFIX + authData + this.GITHUB_HOSTNAME + this.url;
   }
+
+  public async getLatestRelease(): Promise<any> {
```

Comment:
Why have these methods been added to this file?

Reply:
Added them in the past and totally forgot to remove them lol.